src/Umbraco.Web.UI.Client/src/packages/core/property-editor/ui-picker-modal/property-editor-ui-picker-modal.token.ts

```diff
@@ -1,9 +1,6 @@
 import { UmbModalToken } from '@umbraco-cms/backoffice/modal';
 
-export interface UmbPropertyEditorUIPickerModalData {
-	/** @deprecated This property will be removed in Umbraco 15. */
-	submitLabel?: string;
-}
+export type UmbPropertyEditorUIPickerModalData = object;
```

Comment:
I see that I'm guilty of doing this in other places. But it is more ideal to set this to `never`, as object is a bit of a vague type, not really helping the user. where 'never' makes it clear to the user that there is nothing to send here. :-)

Reply:
Okay, this is fine. there is a limit in RoutableModals that enforces this. we will fix that later.